xbmc/windowing/egl/EGLNativeTypeIMX.cpp

```diff
@@ -367,6 +367,19 @@ float CEGLNativeTypeIMX::GetMonitorSAR()
   if(!f_edid)
     return 0;
 
+  // first check if EDID is in binary format by reading 512bytes, compare 1st 8bytes
+  // against EDID 1.4 identificator [0x0,0xff,0xff,0xff,0xff,0xff,0xff,0x0]
+  // if no match, seek to 0 input file and continue with previous method.
+  if (((done = fread(&m_edid, 1, EDID_MAXSIZE, f_edid)) % 128) == 0 && done)
```

Comment:
I guess I have the same question about the & here? but I mean, if it works, then whatever